.github/workflows/regression-test.yaml

```
@@ -48,10 +51,30 @@ jobs:
         run: python -m pip install nox
 
       - name: Run regression test
+        continue-on-error: true
         run: >-
           nox -s regression_tests-${{ matrix.python-version }} -- "$BASE_REF"
-          "$HEAD_REF"
+          "$HEAD_REF" | tee $REGRESSION_TEST_OUTPUT_FILE
```

Usually safer to double escape
```suggestion
          "$HEAD_REF" | tee ${REGRESSION_TEST_OUTPUT_FILE}
```